spot_rl_experiments/experiments/skill_test/test_semantic_gaze.py

```diff
@@ -2,19 +2,90 @@
 # This source code is licensed under the MIT license found in the
```

Comment:
if this is different from the original semantic gaze, then I will suggest that you create a new test script